apps/web/lib/get-posts.ts

```diff
@@ -0,0 +1,119 @@
+import type { BlogCategory, BlogPostListItem } from "@/lib/blog/types";
+import { normalizePages } from "nextra/normalize-pages";
+import { getPageMap } from "nextra/page-map";
+
+// Define our blog post types
+export interface BlogFrontMatter {
+  title: string;
+  date: string;
+  description?: string;
+  tags?: string[];
+  author?: string;
+  category?: string;
```

Comment:
minor but seems like you could avoid all sorts of casting by just making this field be a BlogCategory
```suggestion
  category?: BlogCategory;
```

Reply:
ahh, yes, done!